lib/msf/core/exploit/remote/kerberos/ticket/storage.rb

```diff
@@ -15,9 +15,9 @@ def initialize(info = {})
       super
     end
 
-    # @param [String,nil] protocol The service protocol type, i.e. smb/ldap/winrm/mssql
+    # @param [String] protocol The service protocol type, i.e. smb/ldap/winrm/mssql
     # @return [Array<Msf::OptEnum>]
-    def kerberos_storage_options(protocol: nil)
```

Comment:
if the protocol can be nil won't the option name come out as `::Auth`

Reply:
Protocol is required, this is the old code

Reply:
so it is 😓